llvm/include/llvm/MC/MCTargetOptions.h

```diff
@@ -55,6 +55,7 @@ class MCTargetOptions {
   bool ShowMCEncoding : 1;
   bool ShowMCInst : 1;
   bool AsmVerbose : 1;
+  bool SSE2AVX : 1;
```

Comment:
This may be placed after `X86RelaxRelocations` to make x86-specific options together

Reply:
Done